src/Cli/dotnet/PerformanceLogEventSource.cs

```diff
@@ -0,0 +1,461 @@
+﻿using System;
```

Comment:
nit: few lines in this file is too long. Could you wrap them?

Reply:
Yes, done.